docs/dev-guide/golang/index.md

```diff
@@ -44,6 +44,7 @@ The Developer's guide is divided into the following sections:
    - [Async Activity Completion](/dev-guide/go/features#asynchronous-activity-completion)
    - [Child Workflows](/dev-guide/go/features#child-workflows)
    - [Continue-As-New](/dev-guide/go/features#continue-as-new)
+   - [Schedules](/dev-guide/go/features#schedule-a-workflow)
```

Comment:
Should this be "Schedule a Workflow" so it matches the title? (same for all guides)

Reply:
For maximum SEO, yes—but "Schedules" matches the rest of this list, so I'm inclined to use the shorter form.

Later we can discuss whether to re-title some of the dev-guide pages to make them more consistent.

Reply:
wfm 👍